documents/transfers.md

```diff
@@ -775,6 +775,46 @@ Razorpay::Transfer.fetch(transferId).edit(para_attr)
 
 -------------------------------------------------------------------------------------------------------
 
+### Fetch Reversals for a Transfer
+```rb
+transferId = "trf_JhemwjNekar9Za"
+Razorpay::Transfer.fetch(transferId).reversals
```

Comment:
please remove the method chaining

Reply:
Removed chaining , also updated in doc and unit test

https://github.com/razorpay/razorpay-ruby/blob/3eaf6baa0ef67c9e5b8569ee62d6092e070a1b6b/lib/razorpay/transfer.rb#L35